docs/excel/show-hide-add-in.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Show or hide an Office Add-in in a shared runtime
+description: 'Learn how to programmatically hide or show the UI of an add-in while it runs continuously'
+ms.date: 03/02/2020
+localization_priority: Normal
+---
+
+# Show or hide an Office Add-in in a shared runtime (preview)
+
+Office Add-ins can include any of the following parts:
```

Comment:
What about content add-ins?

Reply:
Content add-ins are not yet supported in shared runtime.